rudder-core/src/main/scala/com/normation/rudder/services/policies/write/PrepareTemplateVariables.scala

```diff
@@ -32,76 +32,115 @@
 *************************************************************************************
 */
 
-package com.normation.cfclerk.services.impl
-
-import com.normation.cfclerk.services._
-import com.normation.cfclerk.domain._
-import com.normation.cfclerk.exceptions._
-import com.normation.stringtemplate.language._
-import com.normation.stringtemplate.language.formatter._
-import java.io._
-import net.liftweb.common._
-import org.antlr.stringtemplate._
-import org.antlr.stringtemplate.language._
-import org.apache.commons.io.{IOUtils,FileUtils}
-import org.joda.time._
-import org.xml.sax.SAXParseException
-import scala.io.Source
-import scala.xml._
-
-import com.normation.utils.Control.sequence
+package com.normation.rudder.services.policies.write
+
+import com.normation.cfclerk.domain.Bundle
+import com.normation.cfclerk.domain.Cf3PromisesFileTemplateId
+import com.normation.cfclerk.domain.PARAMETER_VARIABLE
+import com.normation.cfclerk.domain.SectionVariableSpec
+import com.normation.cfclerk.domain.SystemVariable
+import com.normation.cfclerk.domain.SystemVariableSpec
+import com.normation.cfclerk.domain.Technique
+import com.normation.cfclerk.domain.TechniqueId
+import com.normation.cfclerk.domain.TrackerVariable
+import com.normation.cfclerk.domain.TrackerVariableSpec
+import com.normation.cfclerk.domain.Variable
+import com.normation.cfclerk.exceptions.VariableException
+import com.normation.cfclerk.services.SystemVariableSpecService
+import com.normation.cfclerk.services.TechniqueRepository
+import com.normation.inventory.domain.COMMUNITY_AGENT
+import com.normation.inventory.domain.NOVA_AGENT
+import com.normation.inventory.domain.NodeId
+import com.normation.rudder.domain.reports.NodeConfigId
+import com.normation.rudder.services.policies.BundleOrder
+import com.normation.rudder.services.policies.nodeconfig.NodeConfiguration
+import org.joda.time.DateTime
+import net.liftweb.common.Box
+import net.liftweb.common.EmptyBox
+import net.liftweb.common.Full
+import net.liftweb.common.Loggable
+
+
+trait PrepareTemplateVariables {
```

Comment:
Well, it allows to see with one eyedrop what is the only public interface. And it's on the same file, so I think it's not a big deal.